cleaners/safari.xml

```diff
@@ -19,36 +19,48 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 -->
-<cleaner id="safari" os="windows">
+<cleaner id="safari">
   <label>Safari</label>
   <description>Web browser</description>
   <option id="cache">
     <label>Cache</label>
     <description>Delete the web cache, which reduces time to display revisited pages</description>
-    <action command="delete" search="file" path="$localappdata\Apple Computer\Safari\Cache.db"/>
+      <action command="delete" search="file" path="$localappdata\Apple Computer\Safari\Cache.db"/>
+      <action command="delete" search="file" path="~/Library/Safari/Cache.db"/>
   </option>
   <option id="cookies">
     <label>Cookies</label>
     <description>Delete cookies, which contain information such as web site preferences, authentication, and tracking identification</description>
-    <action command="delete" search="file" path="$APPDATA\Apple Computer\Safari\Cookies\Cookies.plist"/>
-    <action command="delete" search="walk.files" path="$localappdata\Apple Computer\Safari\LocalStorage\"/>
+      <action command="delete" search="file" path="$APPDATA\Apple Computer\Safari\Cookies\Cookies.plist"/>
+      <action command="delete" search="walk.files" path="$localappdata\Apple Computer\Safari\LocalStorage\"/>
+      <action command="delete" search="file" path="~/Library/Safari/Cookies.plist"/>
+      <action command="delete" search="walk.files" path="~/Library/Safari/LocalStorage/"/>
   </option>
   <option id="history">
     <label>History</label>
     <description>List of visited web pages</description>
-    <action command="delete" search="file" path="$APPDATA\Apple Computer\Safari\History.plist"/>
-    <action command="delete" search="file" path="$APPDATA\Apple Computer\Safari\LastSession.plist"/>
```

Comment:
It seems you changed the indentation of lines that otherwise do not need to be touched. :) While this may seem picky, it unnecessarily obscures the code history.

To avoid this, there is a command "make pretty" that formats the XML files with standard indentation.